test/functional/test-viewport.js

```diff
@@ -273,6 +273,15 @@ describe('Viewport', () => {
     bindingMock.expects('getScrollHeight').withArgs().returns(117).once();
     expect(viewport.getScrollHeight()).to.equal(117);
   });
+
```

Comment:
test `updatePaddingBottom`. Put here because it's a function of `viewport` and it's not overridden elsewhere.